src/app/main/component/places/places.component.ts

```diff
@@ -0,0 +1,178 @@
+import { TranslateService } from '@ngx-translate/core';
+import { Component, OnInit, ViewChild, DoCheck } from '@angular/core';
+import { LocalStorageService } from '@global-service/localstorage/local-storage.service';
+import { HttpClient } from '@angular/common/http';
+import { MatDrawer } from '@angular/material';
+import { AgmMarker } from '@agm/core';
+
+import { markers, cards } from './Data.js';
+import { placesIcons } from 'src/app/main/image-pathes/places-icons';
+
+@Component({
+  selector: 'app-places',
+  templateUrl: './places.component.html',
+  styleUrls: ['./places.component.scss']
+})
+export class PlacesComponent implements OnInit, DoCheck {
+  public lat = 49.84579567734425;
+  public lng = 24.025124653312258;
+  public zoom = 13;
+  public cardsCollection: any;
+  public tagList: Array<string> = [];
+  public contentObj: any;
+  public favoritePlaces: Array<number> = [];
+  public stars: Array<string> = ['', '', '', '', ''];
+  public markerList: Array<any> = [];
+  public markerListCopy: Array<any> = [];
+  public searchName = '';
+
+  public redIcon = placesIcons.redIcon;
+  public greenIcon = placesIcons.greenIcon;
+  public bookmark = placesIcons.bookmark;
+  public bookmarkSaved = placesIcons.bookmarkSaved;
+  public star = placesIcons.star;
+  public starHalf = placesIcons.starHalf;
+  public starUnfilled = placesIcons.starUnfilled;
+  public searchIcon = placesIcons.searchIcon;
+  public notification = placesIcons.notification;
+  public share = placesIcons.share;
+
+  public apiKey = 'AIzaSyB3xs7Kczo46LFcQRFKPMdrE0lU4qsR_S4';
```

Comment:
let's move it to enviroment.ts

Reply:
Done 
![image](https://user-images.githubusercontent.com/43093994/119276928-75bc0400-bc25-11eb-8e85-9fdc4de86108.png)